ugvc/scripts/sorter_to_h5.py

```diff
@@ -9,7 +9,7 @@
 import pandas as pd
 
 from ugvc.pipelines.coverage_analysis import generate_stats_from_histogram
-from ugvc.utils.metrics_utils import read_sorter_statistics_csv
+from ugbio_core.sorter_utils import read_sorter_statistics_csv
```

Comment:
read_sorter_statistics_csv was moved to the more specific sorter_utils:
https://github.com/Ultimagen/ugbio-utils/blob/cad8341a927c5641c0d063ace9e5326a2e1ec325/src/core/ugbio_core/sorter_utils.py#L11